changelog/fragments/crds-with-core-types.yaml

```diff
@@ -0,0 +1,5 @@
+entries:
+    - description: >
+        Fix bug that occurs to fill the example in the process to generate the CSV when the Operator API has Kinds named as core types by starting to check its groups. 
+      kind: "bugfix"
+      breaking: false
```

Comment:
```suggestion
      breaking: false
      
```

Reply:
@camilamacedo86  Added the missing newline.